src/plugins/imagery/components/Compass/CompassHUD.vue

```diff
@@ -104,7 +104,10 @@ export default {
         },
         cameraPan: {
             type: Number,
-            required: true
+            required: true,
+            default() {
```

Comment:
If it's a required parameter, then a default value is not needed.

Reply:
@nikhilmandlik to address